src/AbstractArray.php

```diff
@@ -420,6 +420,19 @@ public function containsKey($key)
     }
 
     /**
+     * Find the given value in the array using a closure
+     *
+     * @param \Closure $closure
+     * @param mixed $needle
+     *
+     * @return bool Returns true if the given value is found, false otherwise
+     */
+    public function exists(\Closure $closure, $needle)
```

Comment:
Yes, we should use `callable` instead of `Closure` in favor to https://github.com/bocharsky-bw/Arrayzy/issues/16